selfdrive/ui/sunnypilot/qt/offroad/settings/vehicle/tesla_settings.cc

```diff
@@ -6,9 +6,35 @@
  */
 
 #include "selfdrive/ui/sunnypilot/qt/offroad/settings/vehicle/tesla_settings.h"
+#include "common/params.h"
+#include "common/util.h"
 
 TeslaSettings::TeslaSettings(QWidget *parent) : BrandSettingsInterface(parent) {
+  constexpr int coopSteeringMinKmh = 23; // minimum speed for cooperative steering (enforced by Tesla firmware)
+  Params params;
+  bool is_metric = params.getBool("IsMetric");
+  QString unit = is_metric ? "km/h" : "mph";
+  int display_value;
+  if (is_metric) {
+    display_value = coopSteeringMinKmh;
+  } else {
+    display_value = static_cast<int>(std::round(coopSteeringMinKmh * KM_TO_MILE));
+  }
+  const QString coop_desc = tr("Allows the driver to provide limited steering input while openpilot is engaged. Only works above %1 %2.")
```

Comment:
@sunnyhaibin see discussion here on adding a potential warning about steering oscillations: 

From: https://discord.com/channels/880416502577266699/1413674000252665907/1433150051529785476
To: https://discord.com/channels/880416502577266699/1413674000252665907/1433162157784301688

Reply:
<img width="2236" height="1177" alt="image" src="https://github.com/user-attachments/assets/1dc6ab2b-296c-4a08-82a7-d0744ceee62e" />